rubocop-govuk.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = "rubocop-govuk"
-  spec.version       = "4.4.0"
+  spec.version       = "4.5.0"
```

Comment:
You can drop this line. We'll put together a release after this has merged with the other updates - we follow a bit of a [process on a release](https://github.com/alphagov/rubocop-govuk/blob/main/CONTRIBUTING.md#doing-a-release) to avoid creating releases that are difficult to upgrade too.